internal/api/flags.go

```diff
@@ -5,26 +5,29 @@ import "flag"
 // Flags encapsulates the standard flags that should be added to all commands
 // that issue API requests.
 type Flags struct {
-	dump    *bool
-	getCurl *bool
-	trace   *bool
+	dump               *bool
+	getCurl            *bool
+	trace              *bool
+	insecureSkipVerify *bool
 }
 
 // NewFlags instantiates a new Flags structure and attaches flags to the given
 // flag set.
 func NewFlags(flagSet *flag.FlagSet) *Flags {
 	return &Flags{
-		dump:    flagSet.Bool("dump-requests", false, "Log GraphQL requests and responses to stdout"),
-		getCurl: flagSet.Bool("get-curl", false, "Print the curl command for executing this query and exit (WARNING: includes printing your access token!)"),
-		trace:   flagSet.Bool("trace", false, "Log the trace ID for requests. See https://docs.sourcegraph.com/admin/observability/tracing"),
+		dump:               flagSet.Bool("dump-requests", false, "Log GraphQL requests and responses to stdout"),
+		getCurl:            flagSet.Bool("get-curl", false, "Print the curl command for executing this query and exit (WARNING: includes printing your access token!)"),
+		trace:              flagSet.Bool("trace", false, "Log the trace ID for requests. See https://docs.sourcegraph.com/admin/observability/tracing"),
+		insecureSkipVerify: flagSet.Bool("insecure-skip-verify", false, "Skip validating x.509 certificates against trusted chains"),
```

Comment:
Although X.509 is more technically correct, I think we should put TLS in there instead:

```suggestion
		insecureSkipVerify: flagSet.Bool("insecure-skip-verify", false, "Skip validation of TLS certificates against trusted chains"),
```

Reply:
yeah, I was inspired too much by the error message I was trying to overcome, I guess 😁